gcp/app-engine/main.py

```diff
@@ -1,147 +1,65 @@
-                password=db_password,
-                db=db_name,
-                #ip_type=IPTypes.PRIVATE
-            )
-        else:
-            print('Connecting to vanilla Postgres database')
-            return pg8000.connect(
-                user=db_user,
-                password=db_password,
-                host=db_host,
-                port=db_port,
-                database=db_name
-            )
-
-    pool = sqlalchemy.create_engine(
-        "postgresql+pg8000://",
-        creator=getconn,
-        connect_args={
-            "port": db_port
-        }
-    )
-
-    return pool
+app = Flask(__name__)
 
 @app.route(PUBLIC_API_PREFIX + '/pga-tournaments', methods=['GET'])
 @validate_token
 def get_pga_tournaments(user_email):
     print(f"Got get_pga_tournaments request from user: {user_email}")
-    query = "SELECT data FROM golfbetter.pga_tournaments where 1=1"
-    tournament_id = request.args.get('id', None)
-    if tournament_id is not None:
-        query = query + " and id = '" + str(tournament_id) + "'"
-    pool = get_db_connection()
-    with pool.connect() as db_conn:
-        records = db_conn.execute(sqlalchemy.text(query)).fetchall()
-        return json.dumps([row[0] for row in records], default=json_serial), 200
+    tournament_id = request.args.get('tournamentId', None)
+    db = SessionLocal()
+    try:
+        data = [t.data for t in get_tournaments(db, tournament_id)]
+        schema = TournamentSchema(many=True)
+        return jsonify(schema.dump(data)), 200
+    except Exception as e:
+        print(f'Error querying get_pga_tournaments tournament_id={tournament_id}: {e}', e)
+        return json.dumps({"error": "Internal server error"}), 500
+    finally:
+        db.close()
 
 
 @app.route(PUBLIC_API_PREFIX + '/pga-leaderboard-players', methods=['GET'])
 @validate_token
 def get_pga_leaderboard_players(user_email):
     print(f"Got get_pga_leaderboard_players request from user: {user_email}")
-    query = "SELECT data FROM golfbetter.pga_leaderboard_players where 1=1"
+    id = request.args.get('id', None)
     tournament_id = request.args.get('tournamentId', None)
-    if tournament_id is not None:
-        query = query + " and tournament_id = '" + str(tournament_id) + "'"
     player_id = request.args.get('id', None)
-    if player_id is not None:
-        query = query + " and id = '" + str(player_id) + "'"
-    pool = get_db_connection()
-    with pool.connect() as db_conn:
-        records = db_conn.execute(sqlalchemy.text(query)).fetchall()
-        return json.dumps([row[0] for row in records], default=json_serial), 200
+    db = SessionLocal()
+    try:
+        data = [t.data for t in get_leaderboard_players(db, id, player_id, tournament_id)]
+        schema = PlayerRowV3Schema(many=True)
+        return jsonify(schema.dump(data)), 200
+    except Exception as e:
+        print(f'Error querying get_pga_leaderboard_players id={id} tournament_id={tournament_id} player_id={player_id}: {e}', e)
```

Comment:
I know this is pretty helpful for getting things spun up, but maybe we should move to the logging module sooner rather than later? I'm happy to do that in a separate PR though